$NetBSD: patch-src_crssync_CMakeLists.txt,v 1.9 2024/05/12 19:07:37 gdt Exp $

Fix executing dynamically-linked executable from the build directory.

\todo Understand if this is the right fix, vs some sort of build-time
RPATH command to cmake, and discuss with upstream.

--- src/crssync/CMakeLists.txt.orig	2024-04-19 12:01:44.000000000 +0000
+++ src/crssync/CMakeLists.txt
@@ -21,7 +21,7 @@ else ()
   else()
     add_custom_target(synccrsdb
       COMMENT "Running crssync"
-      COMMAND ${CMAKE_RUNTIME_OUTPUT_DIRECTORY}/crssync
+      COMMAND env LD_LIBRARY_PATH=${CMAKE_RUNTIME_OUTPUT_DIRECTORY}/../lib DYLD_LIBRARY_PATH=${CMAKE_RUNTIME_OUTPUT_DIRECTORY}/../lib ${CMAKE_RUNTIME_OUTPUT_DIRECTORY}/crssync
       DEPENDS crssync
     )
   endif()
